specs/tabs/tabs.md

```diff
@@ -0,0 +1,173 @@
+# fast-tabs
+
+## Overview
+
+*Tabs* are a set of layered sections of content, known as tab panels, that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+
+### Use Cases
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
+  
+### Features
+
+- **Orientation:** Allows the tab list to be oriented vertically to the left or right (depending on language region) of the tab content or horizontally above the content.
+- **Tab activation:** Offers a way to activate a tab and it's content by either focusing a tab or by key press (Space or Enter).
+- **Supplemental content:** Offers a way to add content to the left and/or right of the the tab list.
+
+### Risks and Challenges
+
+*Tabs* has specific guidance on DOM structure by the [WC3](https://w3c.github.io/aria-practices/examples/tabs/tabs-2/tabs.html). This structure lacks logical groupings by separating the tab from its content. Some component libraries compensate for this by creating some kind of intermediary grouping that makes it easier for app authors to implement. Even though the DOM structure disassociates that logical grouping. While other component libraries stick to the DOM structure model. 
+
+### Prior Art/Examples
+- [FAST-DNA Pivot (React)](https://explore.fast.design/components/pivot)
+- [Material UI](https://material-ui.com/components/tabs/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/tabs/)
+- [Ant Design](https://ant.design/components/tabs/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/tabs)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tab-view)
+
+---
+
+### API
+
+*Component name:*
+- `fast-tabs`
+
+*Attributes:*
+- `orientation` - enum
+  - horizontal - default
+  - vertical
+- `activeId` - string
+- `tab-activation` - enum 
+  - focus - default
+  - key-press
+
+*Events:*
+- `onChange` - fires when component `activeId` updates
+
+### Anatomy and Appearance
+
+*Parts:*
+- tabs
+- tab-list
+- tab
+- tab-panel
+
+*Slot Names*
+- before-content
+- after-content
+
+*Template:*
+```HTML
+<div class="tabs" class="tabs">
+    <div class="tab-list" part="tab-list" role="tablist" aria-label="Sample Tabs">
+        <slot name="before-content" part="before-content"></slot>
```

Comment:
How does this work being nested within the role of tablist? Have we tested that this would actually be accessible? As it's being presented here,  it appears as though *anything* can be a `chlid` (or "permitted content" in spec language) which I don't think is the case. Are there examples we can point to in the spec or where this is supported  or permitted?

Reply:
Good point would be easier just to love the slotted content outside the tab list